packages/suite-web/src/support/BrowserDetect/constants.ts

```diff
@@ -0,0 +1,29 @@
+import iconChrome from './images/browser-chrome.png';
+import iconFirefox from './images/browser-firefox.png';
+
+export const supportedBrowsers = [
+    {
+        name: 'Chrome',
+        version: 69,
```

Comment:
Versions in this file do not correspond with versions in `browserslist` file and `supportedBrowsers.ts`. 
Moreover update and download URLs are duplicite with `supportedBrowsers.ts`

Reply:
Browser detect has been reverted (c82d5f6). This is out of scope for this PR.